.nuspec/maui.aotprofile.txt

```diff
@@ -1113,10 +1116,16 @@ Methods:
 	int System.Private.CoreLib.Interop/Sys:PRead (System.Runtime.InteropServices.SafeHandle,byte*,int,long)
 	int System.Random:Next (int,int)
 	int System.Random/XoshiroImpl:Next (int,int)
+	int System.Reflection.Emit.DynamicMethod:AddRef (object)
+	int System.Reflection.Emit.DynamicMethodTokenGenerator:GetToken (System.Reflection.MemberInfo,bool)
+	int System.Reflection.Emit.ILExceptionInfo:LastClauseType ()
+	int System.Reflection.Emit.ILGenerator:get_ILOffset ()
```

Comment:
Hmm, we should see why these appeared. Nothing should use System.Reflection.Emit on startup now.

Do you see a log message like this one?

https://github.com/xamarin/xamarin-android/commit/f5ae4ca41caa7873443ebebe1d0a53595ff6495b

If you have `adb shell setprop debug.mono.log default,timing,assembly,mono_log_level=debug,mono_log_mask=aot` (assembly as one of the categories), it should log this message.

Reply:
`2022-04-15 12:50:41.557 30455-30455/? D/monodroid-assembly: Falling back to System.Reflection.Emit for delegate type '_JniMarshal_PPJ_Z': Boolean n_ContainsItem_J(IntPtr, IntPtr, Int64)`

Reply:
Sounds like that should be fixed in xamarin/android. Just like https://github.com/xamarin/xamarin-android/pull/6855